### ISSUE TITLE
[Ubuntu][netstandardapp1.5] Compiled executable not starting app when there is a dot in folder name

### ISSUE THREAD

**Issue by mmalolepszy** [dotnet/cli#1988]
## Steps to reproduce

1. `mkdir with.dot && cd with.dot`
2. `dotnet new`
3. Edit `project.json` to target `netstandardapp1.5`
4. `dotnet restore`
5. `dotnet build`
6. `./bin/Debug/netstandardapp1.5/ubuntu.14.04-x64/with.dot`

## Expected  behavior

```
Hello World!
```

## Actual behavior

```
Usage: dotnet [--help | app.dll]
```

## Environment data
`dotnet --version` output:

```
.NET Command Line Tools (1.0.0-beta-001933)

Product Information:
 Version:     1.0.0-beta-001933
 Commit Sha:  ede74b6be1

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  14.04
 OS Platform: Linux
 Runtime Id:  ubuntu.14.04-x64
```

## Additional info

- Running `./bin/Debug/netstandardapp1.5/ubuntu.14.04-x64/with.dot ./bin/Debug/netstandardapp1.5/ubuntu.14.04-x64/with.dot.dll` results in expected behavior
- Renaming folder to `withoutdot`, rebuilding and running app again results in expected behavior

**Comment by schellap** [comment]
This should be fixed now... thanks for reporting it!

**Comment by mmalolepszy** [comment]
I confirm it is fixed now on 1.0.0-beta-002202. Thanks!
